kpet/cmd_arch.py

```diff
@@ -35,8 +35,8 @@ def build(cmds_parser, common_parser):
                              help='Limit output to architectures with trees '
                              'matching the REGEX.')
     list_parser.add_argument('regex', nargs='?', default=None,
-                             help='Limit output to architectures '
-                             'matching the regex.')
+                             help='Regular expression fully matching '
+                                  'names of architectures to output.')
```

Comment:
Wouldn't it be good to have a consistent wording between this option and the previous?

Also, I think it reads better if the description is a verb explaining what providing the argument does, like the original "Limit output..." or "Show only...". What do you think?

Reply:
The wording is intentionally different here. The options are normally described in the imperative mood, and arguments are descriptive. E.g. look at the output of `ls --help` or `cp --help`. I suspect this is because arguments are often required so their effect is unavoidable, whereas options are active only if specified. In our case the argument is optional, so it's somewhere in the middle, but it's still indisputably a positional argument rather than an option.

Reply:
Oh, I misunderstood! I thought this was for a `--regex` option. Never mind, it's all good.